release-please-config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "plugins": ["node-workspace"],
+  "bootstrap-sha": "78cc1e225bfc67300cfa9069fce257120fe92a6d",
```

Comment:
this is one before ee6f3e8f55e300df1a75c9be89b47f067bc08dee (`feat: Added --versions option to the code generator (#514)`)  so for now that commit will be included in the changelog processing. I did that because even though that commit bumped things from `21.0.9` to `21.0.10` it did not include any changelog modifications. Again, we can play around with this setting to get the results we want once we see the initial pr release-please creates